pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -78,6 +78,7 @@ func NewMultiOrgAlertmanager(cfg *setting.Cfg, configStore store.AlertingStore,
 			cluster.DefaultProbeTimeout,
 			cluster.DefaultProbeInterval,
 			nil,
+			true,
```

Comment:
@gotjosh in the new version of the alertmanager dependency this signuatre changed. This flag is called `allowInsecureAdvertise`, I think true should be the right value as we don't use TLS, just want to doublecheck with you.

Reply:
That's correct.

Reply:
I tested it to make sure, works fine 👍